herzog/parser.py

```diff
@@ -19,23 +19,23 @@ def __init__(self, cell_type: CellType, lines: Iterable[str]):
         self.lines: List[str] = list()
         for line in lines:
             if CellType.python == self.cell_type:
-                if "pass" == line:
+                if 'pass' == line:
```

Comment:
Please revert this line, and use double quotes throughout to match style with the rest of herzog:
  - double quotes for strings
  - single quotes for keys